docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
-      value: "bos",
-      label: "Boston"
-    },
-    {
-      value: "la",
-      label: "Los Angeles"
-    },
-    {
-      value: "sf",
-      label: "San Francisco"
-    }
-  ];
-  return (
-    <SelectList
-      id="city"
-      name="city"
-      onChange={({ value }) => setCity(value)}
-      options={cityOptions}
-      placeholder="Select city"
-      label="City"
-      value={city}
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use SelectList when the user needs to select from a simple list of items."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'Austin', value: 'austin'},
+    {label:'Los Angeles', value: 'la'},
+    {label:'New York', value: 'nyc'},
+    {label:'San Francisco', value: 'sf'},
+    {label:'Seattle', value: 'seattle'},
+  ]}
+  label='City'
+/>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use SelectList when additional functionality such as subtext or images are needed. Use Dropdown instead."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'List item 1', value: 'Value'},
+    {label:'I want to put an image here, but I can’t :( ', value: 'Value'},
+    {label:'List item 3', value: 'Value'},
+    {label:'List item 4', value: 'Value'},
+    {label:'List item 5', value: 'Value'},
+  ]}
+  label=''
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Order the list items in SelectList either alphabetically or by usage."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'Alberto', value: 'alberto'},
+    {label:'Annie', value: 'annie'},
+    {label:'Ashley', value: 'ashley'},
+    {label:'Ayesha', value: 'ayesha'},
+    {label:'Ryan', value: 'ryan'},
+  ]}
+  label='People who are awesome'
+/>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use SelectList if there are fewer than 4 items in the list and there is space to display all options. Use RadioButtons instead."
+        defaultCode={`
+<SelectList
+  id="selectlistexample"
+  onChange={() => {}}
+  options={[
+    {label:'No', value: 'no'},
+    {label:'Absolutely not', value: 'noagain'},
+  ]}
+  label='Is this a good use of a SelectList?'
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Keep the same type of selection for a group of items. An example of this might be a filter bar. If some items could use SelectList and some items need to use Dropdown, use Dropdown for all the items in the group."
+        defaultCode={`
+<Flex>
```

Comment:
design nit: I'd throw a `gap` of 1-2 in here

Reply:
Done!